modules/underdogmediaBidAdapter.js

```diff
@@ -2,25 +2,27 @@ import * as utils from '../src/utils';
 import { config } from '../src/config';
 import { registerBidder } from '../src/adapters/bidderFactory';
 const BIDDER_CODE = 'underdogmedia';
-const UDM_ADAPTER_VERSION = '1.13V';
+const UDM_ADAPTER_VERSION = '3.0V';
 const UDM_VENDOR_ID = '159';
 
-utils.logMessage(`Initializing UDM Adapter. PBJS Version: ${$$PREBID_GLOBAL$$.version} with adapter version: ${UDM_ADAPTER_VERSION}  Updated 20180604`);
+utils.logMessage(`Initializing UDM Adapter. PBJS Version: ${$$PREBID_GLOBAL$$.version} with adapter version: ${UDM_ADAPTER_VERSION}  Updated 20191028`);
```

Comment:
Adapters are not permitted to reference PREBID_GLOBAL.  For version, please use $prebid.version$